packages/captable/README.md

```diff
@@ -1,4 +1,55 @@
 # Smart contracts for CapTable BRØK
 
-NB! brokProd is still against Arbitrum Goerli
+Smart contracts for CapTable and CapTableRegistry developed in [Solidity v0.8.0](https://docs.soliditylang.org/en/v0.8.0/)
 
+Based on [ERC1400](https://www.shipfinex.com/blog/erc-1400-a-standardized-framework-for-security-tokens), but intended to migrate to [ERC3643](https://www.erc3643.org/) in the future.
+
+The contracts are built and tested with [Hardhat](https://hardhat.org/hardhat-runner/docs/getting-started#overview)
+
+Based on the generated contracts, [tsup](https://www.npmjs.com/package/tsup) is used to generate a TypeScript library, to be used by Brøk API and The Graph configuration.
+
+***NB! brokProd is still against Arbitrum Goerli***
+
+## Project overview
+
+| **Folder** | **Description** |
+|--|--|
+| `artifacts/` | Compiled artifacts generated by Hardhat compile (generated) |
+| `cache/` | The directory used by Hardhat to cache its internal stuff (generated) |
+| `contracts/` | CapTable and CapTableRegistry contracts in Solidity, with dependencies |
+| `deployments/` | Our way to persist smart contract addresses deployed on Arbitrum, generated by [./tasks/generate-deployments.ts](./tasks/generate-deployments.ts) |
+| `dist/` | Generated TypeScript library of CapTable and CapTableRegistry (generated) |
+| `scripts/` | I dont know what this does |
+| `tasks/` | Hardhat tasks to be run with ``hardhat <task name> --network <network name> --log``
+| `test/` | Tests for CapTable and CapTableRegistry |
+| `typechain-types/` | Generated TypeScript bindings for smart contracts (generated) |
+
+
+### .env
+| **Value** | **Description** |
+|--|--|
+| `ETHERSCAN_API_KEY` | API key to Etherscan, used by/in ??? @jon |
```

Comment:
Used for validation of smart contracts on Etherscan. This ensures transparency by verifying that the contract's source code matches the executed bytecode onchain, makes the source code viewable on Etherscan and makes it possible to execute smart contract functions directly from Etherscan.

Reply:
lagt inn redigert versjon av forslaget ditt